kythe/rust/indexer/src/indexer/analyzers.rs

```diff
@@ -0,0 +1,270 @@
+// Copyright 2020 The Kythe Authors. All rights reserved.
```

Comment:
Unit tests, here and elsewhere?

Reply:
I'll definitely add some integration tests for `entries.rs`, but `analyzer.rs` has coverage with the verifier tests in `testdata/` (the `rust_verifier_test` targets in the BUILD file)

Reply:
I'm talking about unit tests here, we normally cover each mod with unit tests, so we can be confident that there are no integration mismatches.

Reply:
Ah, I see what you are saying. I believe I mixed up a non-inline test with an integration test. I have gone ahead and added a unit test file for the entries module. I will add some extra unit tests later today